benchmarks/functional_autograd_benchmark/audio_text_models.py

```diff
@@ -0,0 +1,128 @@
+import torch
+import torch.nn as nn
+
+import torchaudio_models as models
+
+from utils import make_functional, load_weights
+
+def get_wav2letter(device):
```

Comment:
Because PyTorch is Python 3+, I'd love to see some type annotations here (and in the other non-model Python files). Type annotations improve the readability of functions, especially with the helper functions in `utils.py`

e.g.,
```
def get_wav2letter(device: torch.device) -> Tuple[Callable, Tuple]
```